crates/core/src/rpc/surfnet_cheatcodes.rs

```diff
@@ -711,6 +712,14 @@ pub trait SurfnetCheatcodes {
     /// ```
     #[rpc(meta, name = "surfnet_resumeClock")]
     fn resume_clock(&self, meta: Self::Metadata) -> Result<EpochInfo>;
+
+    #[rpc(meta, name = "surfnet_resetAccount")]
+    fn reset_account(
+        &self,
+        meta: Self::Metadata,
+        pubkey_str: String,
+        config: ResetAccountConfig,
```

Comment:
Ahh, to match other RPCs, I think the config will need to be optional `config: Option<ResetAccountConfig>` with defaults set so you can do `let config = config.unwrap_or_default();`